Camera2/app/src/main/java/com/example/ezequiel/camera2/others/Camera2Source.java

```diff
@@ -104,6 +100,7 @@
     private static final SparseIntArray INVERSE_ORIENTATIONS = new SparseIntArray();
     private boolean cameraStarted = false;
     private int mSensorOrientation;
+    private static Detector<?> mDetector;

```

Comment:
I don't think this should be static. There should be a better way to set a new detector.